Sources/POSIX/realpath.swift

```diff
@@ -27,3 +27,31 @@ public func realpath(_ path: String) throws -> String {
     guard let rvv = String(validatingUTF8: rv!) else { throw SystemError.realpath(-1, path) }
     return rvv
 }
+
+private let pathComponentSeparator = "/"
+
+/**
+ Resolves executable, both absolute and relative paths and referred from `PATH` environment variable and
+ return the absolute pathname.
+
+ All components in executable must exists when realpath(executable:) is called.
+*/
+public func realpath(executable: String) throws -> String {
+    if executable.starts(with: pathComponentSeparator) {
+        return try realpath(argv0)
+    }
+    if executable.contains(pathComponentSeparator.first!) {
```

Comment:
We should add some comments explaining what the code is trying to here, i.e., check if the input string is an absolute path or a relative path.